contracts/RegisterDomains.sol

```diff
@@ -0,0 +1,55 @@
+//SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.0;
+
+struct DomainDetails {
+    address controller;
+    uint deposit;
+}
+
+uint256 constant DEPOSIT_PRICE = 1 ether;
+
+contract RegisterDomains {
+    address payable public owner;
+    mapping(string => DomainDetails) domains;
+
+    constructor() {
+        owner = payable(msg.sender);
+    }
```

Comment:
What is the sense to have `owner` in the contract? To say more, why only owner can register the domain? -7

Reply:
fixed: now contract runner is `msg.sender`, have it reflected in tests also by changing runner to `addr1`